source/extensions/filters/http/adaptive_concurrency/concurrency_controller/gradient_controller.cc

```diff
@@ -98,7 +103,18 @@ void GradientController::updateMinRTT() {
     deferred_limit_value_.store(0);
   }
 
-  min_rtt_calc_timer_->enableTimer(config_->minRTTCalcInterval());
+  min_rtt_calc_timer_->enableTimer(
+      applyJitter(config_->minRTTCalcInterval(), config_->jitterPercent()));
+}
+
+std::chrono::milliseconds GradientController::applyJitter(std::chrono::milliseconds interval,
+                                                          double jitter_pct) const {
+  if (jitter_pct == 0) {
+    return interval;
+  }
+
+  const uint32_t jitter_range_ms = interval.count() * jitter_pct;
+  return std::chrono::milliseconds(interval.count() + random_.random() % jitter_range_ms);
```

Comment:
nit: can you add some extra parens here so the order of operations is more clear?